frontend/src/components/RisLawPreview.vue

```diff
@@ -264,18 +284,100 @@ watch(
   },
   { deep: true },
 )
+
+/* -------------------------------------------------- *
+ * Focus handling                                     *
+ * -------------------------------------------------- */
+
+// NB: If we need this in more places in the future, consider extracting
+// this into a composable to reuse it in other places!
+
+const ids = useElementId()
+
+/**
+ * Used for keeping track of the elements that have been made interactive. Needed
+ * for focus handling.
+ */
+let interactiveEls: HTMLElement[] = []
```

Comment:
i needed to think a (short) moment about what Els means as my mind first went to elis

Reply:
Wow I never thought about this but I think that's a habit from the [backbone days](https://backbonejs.org/#View-el) 😆  I see how that can be confusing. Will keep it in mind for next time